numpyro/infer/autoguide.py

```diff
@@ -691,9 +691,10 @@ def median(self, params):
         return self._unpack_and_constrain(loc, params)
 
     def quantiles(self, params, quantiles):
-        transform = self.get_transform(params)
+        posterior = self.get_posterior(params)
+        loc, scale = posterior.loc, jnp.sqrt(posterior.variance)
```

Comment:
It still seems inefficient to use `LowRankMultivariateNormal.__init__()` which computes a bunch of other stuff like `_capacitance_tril`. Why not simply compute (loc, scale) as in Pyro?
```py
loc = params[f'{self.prefix}_loc']
cov_factor = params[f'{self.prefix}_cov_factor']
scale = params[f'{self.prefix}_scale']
scale = scale * jnp.sqrt(jnp.square(cov_factor).sum(-1) + 1)
```

Reply:
Thanks! It is just I was lazy for optimizing this one-time computation. :D